docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -5,110 +5,117 @@ Diffraction Changes
+- The cropping/region of interest selection for Calibration/Focusing is now chosen only on the :ref:`Calibration tab <ui engineering calibration>`, to avoid confusion and duplication of input.
 - The region of interest for Calibration/Focusing can now be selected with a user-supplied custom calibration file.
-- The Focused Run Files input box defaults to the last runs focused on the Focus tab, even if multiple runs were focussed
-- The full calibration setting now has a default value consisting of the path to the ENGINX_full_instrument_calibration_193749.nxs file
-- The usability of the file finder on the Fitting tab has been improved by the addition of file filters based on unit and/or bank
+- The Focused Run Files input box defaults to the last runs focused on the :ref:`Focus tab <ui engineering focus>`, even if multiple runs were focussed
+- The full calibration setting now has a default value consisting of the path to the ``ENGINX_full_instrument_calibration_193749.nxs file``.
+- **The usability of the file finder on the** :ref:`Fitting tab <ui engineering fitting>` **has been improved by the addition of file filters based on unit and/or bank.**
+
+.. image::  ../../images/EngDiff_Fit_Browse_Filters.png
+   :align: center
+   :height: 800px
+
 
 Bugfixes
 ########
-- Sequential fitting in the EngDiff UI now uses the output of the last successful fit (as opposed to the previous fit) as the initial parameters for the next fit.
-- An empty Engineering Diffraction interface is no longer saved if the user saves a project having previously had the interface open at some point in that session
-- The help button on the Engineering Diffraction interface points to the correct page, having been broken in the last release
-- Using the Clear button on the Workspace widget while using the Fitting tab no longer causes issues when you try to load runs back in.
-- On the fitting tab of the EngDiff UI the background can be inspected whether the background subtraction box is checked or not.
+- Sequential fitting in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` now uses the output of the last successful fit (as opposed to the previous fit) as the initial parameters for the next fit.
+- An empty :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` is no longer saved if the user saves a project having previously had the interface open at some point in that session.
+- The help button on the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` points to the correct page, having been broken in the last release.
+- Using the Clear button on the Workspace widget while using the :ref:`Fitting tab <ui engineering fitting>` no longer causes issues when you try to load runs back in.
```

Comment:
Not sure about the use of `issues` here - if we can't be more specific perhaps `errors` - I will try and dif out the PR this relates to.

Reply:
I'm struggling to find the PR this is linked to. The release notes were added as part of #32284 - @jmborr do you know which PR originally fixed the error caused by the Clear button?

Reply:
No, I'm afraid I don't. A lot of stuff was included in  #32284 that I had not coded for and unfortunately I don't know why 😞